manifests/v1beta1/installs/katib-with-kubeflow-cert-manager/ui-virtual-service.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
The same comment as for the `mysql-pvc.yaml` patch.

Reply:
I've implemented `katib-with-kubeflow-cert-manager` to use `katib-with-kubeflow` as a base, which solves needing to duplicate this file.